src/GraphQL.MicrosoftDI/GraphQLBuilder.cs

```diff
@@ -53,7 +53,9 @@ private static MSServiceLifetime TranslateLifetime(ServiceLifetime serviceLifeti
         };
 
     /// <inheritdoc/>
-    public IServiceRegister Configure<TOptions>(Action<TOptions, IServiceProvider>? action = null)
+    public IServiceRegister Configure<
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)]
+    TOptions>(Action<TOptions, IServiceProvider>? action = null)
```

Comment:
```suggestion
    public IServiceRegister Configure<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] TOptions>(Action<TOptions, IServiceProvider>? action = null)
```

Let's to not wrap single line method declaration here and in all other places with new attributes.

Reply:
Often it gets quite unreadable when the method signature goes on forever.  Is there anything else we can do?  I don't mind condensing it some now that `#if` is not required.

Reply:
For me line breaks mixed with generics are more unreadable than a bit longer single line.

Reply:
https://github.com/dotnet/runtime/blob/215b39abf947da7a40b0cb137eab4bceb24ad3e3/src/libraries/System.Private.CoreLib/src/System/Reflection/TypeDelegator.cs#L30

https://github.com/dotnet/runtime/blob/eb9dd67ce9fc64c468cf5c795fda7057310c22fc/src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilder.cs#L391

Result of briefly looking through MS code. They use attribute on the same line. Also I saw in some of their code aliases for attribute argument to shorten the length. We may use aliases even for the single parametet value. The same may be done for attribute name I think to shorten the overall string even more.

Reply:
```suggestion
    public IServiceRegister Configure<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] TOptions>(Action<TOptions, IServiceProvider>? action = null)
```